src/ga/map.js

```diff
@@ -6,6 +6,7 @@ goog.require('goog.ui.Menu');
 goog.require('goog.ui.MenuItem');
 goog.require('goog.dom');
 goog.require('goog.dom.classlist');
+goog.require('goog.html.SafeHtml');
```

Comment:
Does it works without this require ?

Reply:
No it does not. When I run the command `make -f Makefile-ga build-ga` I get the following error:
`
ERR! compile /home/ltpin/geoadmin-ol3/src/ga/map.js:246: ERROR - missing require: 
'goog.html.SafeHtml'
ERR! compile   this.geocoderDialog_.setSafeHtmlContent(goog.html.SafeHtml.create('div', {'id': 'geocoderList'}));
ERR! compile                                         
ERR! compile
ERR! compile 1 error(s), 0 warning(s)
ERR! compile
ERR! Process exited with non-zero status, see log for more detail: 1
Makefile-ga:14: recipe for target 'build/ga.js' failed
`